src/components/SettingsDialog/Components/Footer.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/constants/style";
+@import "../SettingsDialog";
+
+// settings-dialog container
+// $settings-dialog-container--sides-small: 32px;
+// $settings-dialog-container--sides-large: 92px;
+// $settings-dialog-container--bottom: 62px;
+
+.container {
```

Comment:
I think you should use a more specific name instead of the generic `container` name

Reply:
Updated the naming. Thank you.